### DISCUSSION TITLE
Template is not working

### DISCUSSION THREAD

**Discussion by m2017atTR** [Dotnet-Boxed/Templates#165] [Q&A]
In visual studio 2017 and 2015 template not shown in list anymore

**Comment by timematcher** [comment]
@m2017atTR Please make sure that in **Visual Studio 2017**,  **.NET 4.6.1** is selected in the dropdown and corresponding language **Visual C#** or **Visual Basic** is selected in the tree in the templates window
![image](https://cloud.githubusercontent.com/assets/11568346/24982809/da8b9e96-1ffc-11e7-810b-cd4ddc1a372e.png)

**Comment by RehanSaeed** [comment]
I had no idea you had to do that. Isn't 4.6.1 selected by default anyway?

**Comment by m2017atTR** [comment]
I think coused by something else, now i can see and its running.
Using VS 2017
thanks.
